doto-backend/package.json

```diff
@@ -1,27 +1,32 @@
 {
-    "name": "doto-backend",
-    "version": "1.0.0",
-    "description": "",
-    "main": "index.js",
-    "scripts": {
-        "test": "mocha",
-        "start": "nodemon ./src/index.js"
-    },
-    "keywords": [],
-    "author": "",
-    "license": "ISC",
-    "dependencies": {
-        "body-parser": "^1.19.0",
-        "cors": "^2.8.5",
-        "dotenv": "^8.2.0",
-        "express": "^4.17.1",
-        "jsonwebtoken": "^8.5.1",
-        "mocha": "^7.1.0",
-        "mongoose": "^5.9.4",
-        "mongoose-unique-validator": "^2.0.3",
-        "nodemon": "^2.0.2",
-        "passport": "^0.4.1",
-        "passport-google-oauth20": "^2.0.0",
-        "swagger-ui-express": "^4.1.3"
-    }
```

Comment:
I haven't worked with the backend, so am not sure. I used npm update when my app crashed to fix launching the server

Reply:
> Could I have some clarification as to the purpose of this PR? I'm unsure what issue 'travel time' is meant to solve, and the linked issue appears to already be fixed in `master` as tasks cannot be created with Due Dates before the time of form submission.

So, the travel time is to account for travelling between tasks. The scheduler is lazy in the sense it puts a task right after the latest one available causing an overlap. This can definitely be improved upon with GPS but might introduce complex algorithms.